MonoGame.Framework/Graphics/SpriteBatchItem.cs

```diff
@@ -19,70 +19,77 @@ public SpriteBatchItem ()
             vertexBR = new VertexPositionColorTexture();            
 		}
 		
-		public void Set ( float x, float y, float w, float h, Color color, Vector2 texCoordTL, Vector2 texCoordBR )
+		public void Set ( float x, float y, float w, float h, Color color, Vector2 texCoordTL, Vector2 texCoordBR, float depth )
 		{
 			vertexTL.Position.X = x;
```

Comment:
> The  Sort  moves object references (pointers) around which are the size of a register (32/64bit), but then, there are also cache misses when it access the  SortKey 

I wonder if adding a structure like:
```
struct SpriteBatchSortItem
{
   int index;
   float SortKey;
}
```
... then doing the sort on an array of that structure would be better for sort performance.  We can then also change `SpriteBatchItem` to a structure as we're not sorting them directly.